sechub-webui/src/main/java/com/mercedesbenz/sechub/webui/StatusController.java

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.webui;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import reactor.core.publisher.Mono;
+
+@Controller
+public class StatusController {
+    @Autowired
+    StatusService statusService;
+
+    @Autowired
+    SecHubServerAccessService accessService;
+
+    @GetMapping("/status")
+    String status(Model model) {
+        Mono<String> secHubServerVersion = statusService.getServerVersion();
+
+        model.addAttribute("sechubServerUrl", accessService.getSecHubServerUrl());
```

Comment:
do we really need here camel case attributes? I am not sure how spring webflux handles it (now and in future) - I would say it make no difference, but...

Reply:
The model values are used inside the Thymeleaf templates. It looks strange to not use camel case, especially, because it is possible to pass objects to the model as well.